### ISSUE TITLE
query generic http endpoint

### ISSUE THREAD

**Issue by DeoLeung** [nineinchnick/trino-rest#180]
Hi,

just wonder if this connector could be used to do generic http request?

maybe simply return the status code , content type, headers and content and leave the rest to trino's [json](https://trino.io/docs/current/functions/json.html)

**Comment by nineinchnick** [comment]
No, it cannot be used for this. It would make writing queries a lot harder if you'd want to fetch multiple pages of data, handle predicate pushdown, etc.
